calculator.rb

```diff
@@ -0,0 +1,75 @@
+
+data = [
+  4,
+  "6",
+  { result: "Deset"}
+]
+
+
+puts "#{data[0]} + #{data[1]} = #{data[2][:result]} "
+
+
+
+
```

Comment:
please add class inside specific directory such as

lib/classes/calculator.rb

then inside new file such as  lib/run.rb use require to import class
require "lib/classes/calculator.rb

it is practice in programming that each class is placed in specific file so that it can be used easily over multiple places in code . 

https://www.thoughtco.com/requre-method-2908199